test/Shared/UserManagerTestBase.cs

```diff
@@ -1050,41 +1051,43 @@ public Task<IdentityResult> ValidateAsync(UserManager<TUser> manager, TUser user
             roleClaims = await manager.GetClaimsAsync(role);
             Assert.Equal(0, roleClaims.Count);
 
-            IdentityResultAssert.VerifyRoleManagerSuccessLog(manager.Logger, "RemoveClaimAsync", role.Id.ToString());
+            IdentityResultAssert.VerifyRoleManagerSuccessLog(manager.Logger, "RemoveClaimAsync", await manager.GetRoleIdAsync(role));
         }
 
         [Fact]
         public async Task CanRoleFindById()
         {
             var manager = CreateRoleManager();
-            var role = CreateRole("FindByIdAsync");
-            Assert.Null(await manager.FindByIdAsync(role.Id.ToString()));
+            var role = CreateTestRole("FindByIdAsync");
```

Comment:
Also it doesn't make sense for CreateTestUser and CreateTestRole not to look the same, for the same reason CreateTestUser's was done the way it was, CreateTestRole should follow the same pattern